src/components/Courses/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '../../../../common/Button/Button';
+import Input from '../../../../common/Input/Input';
+import {
+	BUTTON_TEXT_ADD_COURSE,
+	BUTTON_TEXT_SEARCH,
+} from '../../../../constants';
+import styles from './searchBar.module.scss';
+
+const SearchBar = () => {
+	return (
+		<div className={styles.wrapper}>
+			<form
+				className={styles.form}
+				onSubmit={(event) => {
+					event.preventDefault();
+				}}
+			>
+				<Input></Input>
```

Comment:
Using a label tag for input is considered good practice

Reply:
fixed, thanks